package.json

```diff
@@ -118,6 +117,7 @@
     "rimraf": "^2.6.1",
     "satisfied": "^1.1.1",
     "screener-runner": "^0.10.30",
+    "shallowequal": "^1.1.0",
```

Comment:
Is `shallowequal` used for distribution purpose? Or internal?

Reply:
It's used only in docs, see https://github.com/stardust-ui/react/pull/928#discussion_r258100901